notebooks/multimodal_agent_with_fastrag_haystack.ipynb

```diff
@@ -0,0 +1,2703 @@
+{
```

Comment:
<p>can we show the original image to confirm the correctness of the response?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepset-ai/haystack-cookbook/pull/149/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>